### PR TITLE
Added screens and navigation for need permission on docs

### PR DESCRIPTION
**Pull request by SasankAchyuta9**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/applicant/upload-document/need-permission/content.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+export const generateContent: TranslationFn = content => {
+  const translations = languages[content.language]();
+  const request = content.additionalData?.req;
+  const userCase = request.session.userCase;
+  const caseId = userCase.id as string;
+  const caseType = userCase.caseTypeOfApplication;
+  return {
+    ...translations,
+    form: {
```
Comment (vivek-sekhar) [inline src/main/steps/applicant/upload-document/need-permission/content.ts:52]: pls use object.assign above return

File: src/main/steps/applicant/applicantCaseSequence.ts [hunk 2]
```diff
@@ -385,7 +386,15 @@ export const applicantCaseSequence: Step[] = [
...
     url: APPLICANT_UPLOAD_DOCUMENT_LIST_START_URL,
     showInSection: Sections.AboutApplicantCase,
-    getNextStep: () => APPLICANT_UPLOAD_DOCUMENT_LIST_SUMMARY_URL,
+    getNextStep: caseData =>
+      caseData.start === YesOrNo.NO
+        ? APPLICANT_UPLOAD_DOCUMENT_LIST_NEED_PERMISSION_URL
+        : APPLICANT_UPLOAD_DOCUMENT_LIST_SUMMARY_URL,
+  },
+  {
+    url: APPLICANT_UPLOAD_DOCUMENT_LIST_NEED_PERMISSION_URL,
```
Comment (vivek-sekhar) [inline src/main/steps/applicant/applicantCaseSequence.ts:395]: url shd be permission_to_submit_extra_evidence

File: src/main/steps/respondent/upload-document/need-permission/content.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import { TranslationFn } from '../../../../app/controller/GetController';
```
Comment (vivek-sekhar) [inline src/main/steps/respondent/upload-document/need-permission/content.ts:1]: folder name shd be submit-extra-evidence

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/respondent/upload-document/need-permission/content.ts (hunk 3 above)
Comment (SasankAchyuta9) [reply src/main/steps/respondent/upload-document/need-permission/content.ts:1]: done this.

File: src/main/steps/applicant/upload-document/need-permission/content.ts (hunk 1 above)
Comment (SasankAchyuta9) [reply src/main/steps/applicant/upload-document/need-permission/content.ts]: done this.

File: src/main/steps/respondent/upload-document/submit-extra-evidence/content.ts [hunk 4]
```diff
@@ -44,14 +44,10 @@ export const generateContent: TranslationFn = content => {
   const request = content.additionalData?.req;
   const userCase = request.session.userCase;
   const caseId = userCase.id as string;
+  const cancelLink = `${RESPONDENT_TASK_LIST_URL}/${caseId}`;
```
Comment (vivek-sekhar) [inline src/main/steps/respondent/upload-document/submit-extra-evidence/content.ts:47]: No need of varaible

File: src/main/steps/respondent/respondentcaseSequence.ts [hunk 5]
```diff
@@ -543,7 +543,7 @@ export const respondentCaseSequence: Step[] = [
     getNextStep: () => RESPONDENT_TASK_LIST_URL,
   },
   {
-    url: RESPONDENT_UPLOAD_DOCUMENT_LIST_NEED_PERMISSION_URL,
+    url: RESPONDENT_UPLOAD_DOCUMENT__PERMISSION_TO_SUBMIT_EXTRA_EVIDENCE,
```
Comment (vivek-sekhar) [inline src/main/steps/respondent/respondentcaseSequence.ts:546]: double underscore? remove it

File: src/main/steps/applicant/upload-document/submit-extra-evidence/content.ts [hunk 6]
```diff
@@ -47,17 +47,11 @@ export const generateContent: TranslationFn = content => {
   const userCase = request.session.userCase;
   const caseId = userCase.id as string;
   const caseType = userCase.caseTypeOfApplication;
+  const cancelLink =
```
Comment (vivek-sekhar) [inline src/main/steps/applicant/upload-document/submit-extra-evidence/content.ts:50]: No need of extra variable

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/applicant/upload-document/submit-extra-evidence/content.ts (hunk 6 above)
Comment (SasankAchyuta9) [reply src/main/steps/applicant/upload-document/submit-extra-evidence/content.ts:50]: Done this.

File: src/main/steps/respondent/respondentcaseSequence.ts (hunk 5 above)
Comment (SasankAchyuta9) [reply src/main/steps/respondent/respondentcaseSequence.ts:546]: Done this.

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**